rllib/utils/metrics/metrics_logger.py

```diff
@@ -32,6 +31,20 @@ class MetricsLogger:
     data, such as sliding windows or EMA coefficients.
     - Optionally clearing all logged values after a `reduce()` call to make space for
     new data.
+    - Tracking throughputs of logged values.
+
+    We mostly use this to aggregate stats from across components in a tree structure.
+    For example, we aggregate stats from across parallel EnvRunners like so:
+
+        Root        (Driver/Algorithm object)
+        ┌─┴─┐
+      A1    A2      (AggregatorActor)
+    ┌─┴─┐  ┌─┴─┐
+    E1 E2  E3  E4   (EnvRunner)
+
+    Here, the Driver will call MetricsLogger.compile() to get the aggregated stats once per
+    Algorithm iteration and reports them. If you want to log custom metrics, you can do so by
+    calling `log_value` or `log_dict` with the appropriate arguments on an existing MetricsLogger.
```

Comment:
Putting this here to make it clear how we envision MetricsLogger's role in RLlib.
I believe the "tree aggregation" aspect has not been clear enough, judging from conversations I had about this.